src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -342,3 +423,629 @@ def test_WINNF_FT_S_PCR_1(self, config_filename):
+        'measCapability': device_b['measCapability']
+    }
+    conditionals = [conditionals_b]
+    del device_b['installationParam']
+    del device_b['cbsdCategory']
+    del device_b['airInterface']
+    del device_b['measCapability']
+
+    # Create the actual config.
+    devices = [device_a, device_b]
+    config = {
+        'registrationRequests': devices,
+        'conditionalRegistrationData': conditionals,
+        'palRecords': pal_records
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_PCR_5_default_config)
+  def test_WINNF_FT_S_PCR_5(self, config_filename):
+    """Unsuccessful PPA Creation with one or more CBSDs Outside Service Area.
+
+    Checks SAS UUT rejects creation of a PPA boundary if at least one of
+    the CBSDs included in the CBSD cluster list does not belong to the PAL holder.
+    """
+    # Load the Config file.
+    config = loadConfig(config_filename)
+
+    # Inject the PAL records.
+    for pal_record in config['palRecords']:
+      self._sas_admin.InjectPalDatabaseRecord(pal_record)
+
+    # Register devices and check the response.
+    cbsd_ids = self.assertRegistered(config['registrationRequests'],
+                                     config['conditionalRegistrationData'])
+
+    # Prepares the PAL Ids to trigger ppa creation request.
+    pal_ids = [record['palId'] for record in config['palRecords']]
+
+    # Create PPA creation request to SAS UUT.
+    ppa_creation_request = {
+        "cbsdIds": cbsd_ids,
+        "palIds": pal_ids
+    }
+
+    # Trigger PPA Creation to SAS UUT and expect the failure response.
+      # SAS does not create a PPA and generates an error.
+    self.assertPpaCreationFailure(ppa_creation_request)
+
+  def generate_PCR_6_default_config(self, filename):
+    """Generate the WinnForum configuration for PCR 6."""
+    # File path where SAS UUT claimed ppa boundary generated in PCR.1 test
+    pcr_1_test_config_file_path = os.path.join('testcases', 'configs',
+                                               'test_WINNF_FT_S_PCR_1',
+                                               'default.config')
+    sas_uut_claimed_ppa_boundary_file_path = getSasUutClaimedPpaBoundaryFilePath(
+        'default.config')
+
+    # Load SAS UUT claimed ppa boundary and check if any error while retrieving
+    # SAS UUT claimed ppa boundary generated in PCR.1 test.
+    try:
+      with open(sas_uut_claimed_ppa_boundary_file_path, 'r') as claimed_ppa_file:
+        user_claimed_ppa_contour = json.load(claimed_ppa_file)
+    except IOError:
+      raise RuntimeError('ConfigError:There is an error in reading path:%s \n\n'
+                         % sas_uut_claimed_ppa_boundary_file_path)
+
+    # Expand the user claimed ppa boundary by 1 kilometer.
+    user_claimed_ppa_contour_shapely = utils.ToShapely(
+      user_claimed_ppa_contour['features'][0]['geometry']).buffer(1e-2)
+    user_claimed_ppa_contour_geometry = json.loads(utils.ToGeoJson(
```

Comment:
Better to use the parameter `as_dict` of the ToGeoJson routine.
So if you want to get a dict you do:
```
  user_claimed_ppa_contour_geometry = utils.ToGeoJson(used_claimed_ppa_contour_shapely, 
                                                 as_dict=True) 
```

Reply:
Added   the as_dict=True instead of json.loads`